gateway-package/src/main/resources/bin/start.sh

```diff
@@ -92,7 +112,7 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${GATEWAY_CODE} java \
     -Denvironment.ipAddress=${ZOWE_IP_ADDRESS} \
     -Dapiml.gateway.timeoutMillis=${APIML_GATEWAY_TIMEOUT_MILLIS} \
     -Dapiml.security.ssl.verifySslCertificatesOfServices=${VERIFY_CERTIFICATES} \
-    -Dapiml.security.auth.zosmfServiceId=zosmf \
+    -Dapiml.security.auth.zosmfServiceId=mockzosmf \
```

Comment:
Won't this overwrite the value for the production run?

Reply:
good catch, this needs to be parametrized, I added new parameter with default value zosmf